pkg/query/args.go

```diff
@@ -276,7 +277,7 @@ func (a *Args) Prepare(writers ...io.Writer) (*Statement, error) {
 		s.Output = io.MultiWriter(writers...)
 	}
 
-	s.Query = goDB.NewQuery(queryAttributes, queryConditional, s.HasAttrTime, s.HasAttrIface)
+	s.Query = goDB.NewQuery(queryAttributes, queryConditional, s.HasAttrTime, s.HasAttrIface).LowMem(a.LowMem)
```

Comment:
Not urgent, but I'll refactor this to accept functional options as part of #43

Reply:
Sure thing, no strong feelings here. I'm still undecided which I like more, functional options are chaining... :-P